src/validations/Student.js

```diff
@@ -32,6 +32,8 @@ const salary_offered = Yup.number().nullable().required("Salary Offered is requi
                           .min(0, 'Min value 0.')
                           .max(1000000, 'Salary should be in range of between 0 to 10 Lakh.');
 
+const number_of_internship_hours = Yup.number().required("Intership Hour is required.").positive().integer()
```

Comment:
```suggestion
const number_of_internship_hours = Yup.number().required("Internship hours are required.").positive().integer()
```

Reply:
Done with this change as mentioned above.